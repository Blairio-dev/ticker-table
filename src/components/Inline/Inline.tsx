import "./Inline.scss";

interface InlineProps {
  nodes: React.ReactNode[];
}

const Inline = ({ nodes }: InlineProps) => (
  <div className="Inline-wrapper">
    {nodes.map((node, index) => (
      <div key={index}>{node}</div>
    ))}
  </div>
);

export { Inline };
