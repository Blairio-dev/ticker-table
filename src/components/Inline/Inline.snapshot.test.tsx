import React from "react";
import renderer from "react-test-renderer";
import { Inline } from "./Inline";

const DEFAULT_PROPS = {
  nodes: [<div>Left node</div>, <div>Right node</div>],
};

describe("DataTable Snapshots", () => {
  it("Renders with default props", () => {
    const tree = renderer.create(<Inline {...DEFAULT_PROPS} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
