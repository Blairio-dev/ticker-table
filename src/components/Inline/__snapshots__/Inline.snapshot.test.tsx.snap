// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable Snapshots Renders with default props 1`] = `
<div
  className="Inline-wrapper"
>
  <div>
    <div>
      Left node
    </div>
  </div>
  <div>
    <div>
      Right node
    </div>
  </div>
</div>
`;
